plugins/kn/README.md

```diff
@@ -0,0 +1,17 @@
+# kn - Knative CLI
+
+This plugin provides autocompletion for [kn](https://knative.dev/docs/install/install-kn/) revison, route, service, source, and trigger operations.
```

Comment:
This URL doesn't work anymore, can you fix that?

Reply:
please link https://knative.dev/docs/install/client/#kn